doc/source/user_guide/tutorial_parallelization.rst

```diff
@@ -61,3 +61,13 @@ The number of jobs can be specified.
         Parallel(n_jobs=4)(delayed(task)(i) for i in pics)
 
     %timeit joblib_loop()
+
+
+``apply_parallel`` Maps a function in parallel across an array.
+
+.. code-block:: python
+    
+    from skimage.util.apply_parallel import apply_parallel
+    apply_parallel(threshold_local, a, chunks=(6, 6), depth=5,
+                   extra_arguments=(3,),
+                   extra_keywords={'mode': 'reflect'})
```

Comment:
It is a good addition!
I think `apply_parallel` should be mentioned before `joblib`. It would be also nice to have more detailed description what `apply_parallel` does in this case, and execution time measurements (for comparison to `joblib`).

Reply:
Sure thanks for review @soupault 
I will do the first part.
>  It would be also nice to have more detailed description what apply_parallel does in this case, and execution time measurements (for comparison to joblib).

If you can explain how I can do above if would be very helpful.

Reply:
Sure thanks for review @soupault
I will do the first part.

> It would be also nice to have more detailed description what apply_parallel does in this case, and execution time measurements (for comparison to joblib).

If you can explain how I can do above if would be very helpful.